src/components/editor/EditorAppController.jsx

```diff
@@ -23,13 +25,37 @@ export default class EditorAppController extends BaseComponent {
     }
   }
 
+  isrestarted (){
+    http.get('/isrestarted', (response) => {
+      let signal = '';
+      response.on('data', (chunk) => {
+        signal += chunk;
+      });
+      response.on('end', () => {
+          if (signal === 'false'){
+              window.alert("Servers restarted successfully");
+            }
+          else if (signal === 'true'){
+              window.alert('Pinging Servers');
+              this.isrestarted();
+            }
+          else{
+              window.alert('error');
```

Comment:
As we could expect errors in the requests while the server is down during restart I feel like this is not the best method. I think it would be better to, if the 5 pings or whatever the limit is that I talked about above runs out then we say there was an error, does that sound okay?